src/org/exist/storage/NativeValueIndex.java

```diff
@@ -1323,7 +1323,7 @@ private AtomicValue convertToAtomic( int xpathType, String value )
                 atomic = new StringValue( value ).convertTo( xpathType );
             }
             catch( final XPathException e ) {
-                LOG.warn( "Node value '" + value + "' cannot be converted to " + Type.getTypeName( xpathType ) );
+                LOG.debug( "Node value '" + value + "' cannot be converted to " + Type.getTypeName( xpathType ) );
```

Comment:
The change on this line doesn't make sense to me. This I believe is a genuine problem which we are then effectively hiding from the user. This should at least be a WARN IMHO. @wolfgangmm @dizzzz Can you please explain why you think this should be hidden during normal operation?